packages/amplication-git-pull-request-service/src/core/diff/diff.service.ts

```diff
@@ -43,52 +48,35 @@ export class DiffService {
       'Cant get the same build id'
     );
 
-    return this.firstBuild(newBuildPath);
-    // // return all the new files if an old build folder dont exist
-    // if (existsSync(oldBuildPath) === false) {
-    //   return this.firstBuild(newBuildPath);
-    // }
+    DiffService.assertBuildExist(oldBuildPath);
+    DiffService.assertBuildExist(newBuildPath);
 
-    // DiffService.assertBuildExist(newBuildPath);
+    this.logger.info({ oldBuildPath, newBuildPath });
```

Comment:
This logger message is meaningless. Please add some human-readable text so we can understand what this means.

For example,
```ts
this.logger.info('This thing does something', { oldBuildPath, newBuildPath });
```